ansible/kayobe-automation-run-tempest.yml

```diff
@@ -4,16 +4,14 @@
   gather_facts: false
   vars:
     results_path_local: "{{ lookup('env', 'PWD') }}"
-    rally_image: 'gwee/rally'
+    rally_image: 'stackhpc/docker-rally'
     rally_tag: latest
     rally_image_full: "{{ rally_image }}:{{ rally_tag }}"
     rally_no_sensitive_log: true
     # This ensures you get the latest image if the image is updated
     # and the tag isn't e.g when using the latest tag.
     rally_force_pull: true
-    rally_docker_registry: "{{ kolla_docker_registry }}"
```

Comment:
Possibly default these to none? And leave the login logic below?
```
rally_docker_registry_username:
rally_docker_registry_password:
``` In case someone wants to use an alternative image that is behind a login?